pkgs/development/python-modules/stups-pierone/default.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, requests
+, stups-cli-support
+, stups-zign
+, pytest
+, pytestcov
+, isPy3k
+}:
+
+buildPythonPackage rec {
+  pname = "stups-pierone";
+  version = "1.1.45";
+  disabled = !isPy3k;
+
+  src = fetchFromGitHub {
+    owner = "zalando-stups";
+    repo = "pierone-cli";
+    rev = version;
+    sha256 = "1ggfizw27wpcagbbk15xpfrhq6b250cx4278b5d7y8s438g128cs";
+  };
+
+  propagatedBuildInputs = [
+    requests
+    stups-cli-support
+    stups-zign
+  ];
+
+  preCheck = "
+    export HOME=$TEMPDIR
+  ";
+
+  checkInputs = [
+    pytest
+    pytestcov
```

```suggestion
    pytestcov
    hypothesis
```

to match their setup.py
```
tests_require=['pytest-cov', 'pytest', "hypothesis"],
```

```
  ERROR: Could not find a version that satisfies the requirement hypothesis (from versions: none)
  ERROR: No matching distribution found for hypothesis
  error: Command '['/nix/store/pdq7cyv9ifhrmlshfvyd4cy61dvigicw-python3-3.8.1/bin/python3.8', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/build/tmp68vcucek', '--quiet', 'hypothesis']' returned non-zero exit status 1.
cannot build derivation '/nix/store/7gmwx297pck2xm3m0i78dcl8a65rh1kv-env.drv': 2 dependencies couldn't be built
[1 built (2 failed), 0.0 MiB DL]
error: build of '/nix/store/7gmwx297pck2xm3m0i78dcl8a65rh1kv-env.drv' failed
https://github.com/NixOS/nixpkgs/pull/77521
2 package failed to build:
python37Packages.stups-pierone python38Packages.stups-pierone
```